web/src/styles/wiki.css

```
@@ -1,30 +1,39 @@
-  --wk-amber-banner:    var(--olive-100);
-  --wk-code-bg:         var(--bg-warm);
-  --wk-display:    var(--font-sans);
-  --wk-body-serif: 'Source Serif 4', ui-serif, Georgia, serif;
-  --wk-chrome:     var(--font-sans);
-  --wk-mono:       var(--font-mono);
+  --wk-amber: var(--olive-500);
+  --wk-amber-bg: var(--olive-200);
+  --wk-amber-banner: var(--olive-100);
+  --wk-code-bg: var(--bg-warm);
+  --wk-display: var(--font-sans);
+  --wk-body-serif: "Source Serif 4", ui-serif, Georgia, serif;
```

Stylelint (17.9.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)